### PR TITLE
PROF-8999: Describe timelines view for Node.js

### PR DESCRIPTION
**Pull request by szegedi**
### What does this PR do? What is the motivation?
Timelines for Node.js are in public beta, we should document them for users.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes
For most languages, the text in respective language tab for timelines in `profile_visualizations` and `connect_traces_and_profiles` is the same. Here, the text in the latter does not have description of one kind of lanes ("asynchronous activity".) This is intentional, as sadly we can't associate those activities with a particular trace, so they don't show there.

### CODE REVIEW

File: content/en/profiler/connect_traces_and_profiles.md [hunk 1]
```diff
@@ -213,6 +216,13 @@ Each lane represents a **thread**. Threads from a common pool are grouped togeth
 
 Lanes on top are runtime activities that may add extra latency. They can be unrelated to the request itself.
 {{< /programming-lang >}}
+{{< programming-lang lang="nodejs" >}}
+See [prerequisites](#prerequisites) to learn how to enable this feature for Node.js.
+
+There is one lane for the JavaScript **thread**.
+
+Lanes on the top are **runtime activities** that may add extra latency to your request, currently these are garbage collector activities.
```
Comment (buraizu) [inline content/en/profiler/connect_traces_and_profiles.md:224]:
```suggestion
Lanes on the top are garbage collector **runtime activities** that may add extra latency to your request.
```

File: content/en/profiler/profile_visualizations.md [hunk 2]
```diff
@@ -137,6 +137,18 @@ The thread ID is shown as `native-thread-id (ruby-object-id)` where the native t
...
 **Note**: The Ruby VM or your operating system might reuse native thread IDs.
 
+[1]: /profiler/connect_traces_and_profiles/#prerequisites
+{{< /programming-lang >}}
+{{< programming-lang lang="nodejs" >}}
+See [prerequisites][1] to learn how to enable this feature for Node.js.
+
+There is one lane for the JavaScript **thread**.
+
+There can also be lanes visualizing various kinds of **asynchronous activity**, currently these are DNS requests and TCP connect operations. The number of lanes matches
```
Comment (buraizu) [inline content/en/profiler/profile_visualizations.md:147]:
```suggestion
There can also be lanes visualizing various kinds of **asynchronous activity** consisting of DNS requests and TCP connect operations. The number of lanes matches
```

File: content/en/profiler/profile_visualizations.md [hunk 3]
```diff
@@ -137,6 +137,18 @@ The thread ID is shown as `native-thread-id (ruby-object-id)` where the native t
...
+the maximum concurrency of these activities so they can be visualized without overlaps.
+
+Lanes on the top are **runtime activities** that may add extra latency to your request. Currently these are various garbage collection activities.
```
Comment (buraizu) [inline content/en/profiler/profile_visualizations.md:150]:
```suggestion
Lanes on the top are garbage collector **runtime activities** that may add extra latency to your request.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Looks good overall, just left some minor wording updates per the [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense)

**Comment by szegedi** [comment]
> Looks good overall, just left some minor wording updates per the documentation style guide

Awesome; I was unhappy with the wording there myself, but couldn't hit on the shape that'd feel right. Your suggestions certainly do. Thank you @buraizu!

**Review by buraizu — APPROVED** [review]

**Pull request merged**